fluent-syntax/src/parser.js

```diff
@@ -685,13 +687,15 @@ class FluentParser {
 
     if (ps.isIdentifierStart()) {
       const id = this.getIdentifier(ps);
+      ps.peekBlank();
```

Comment:
alternatively, you could store the ptr, `skipBlank` and then only take `currentChar === "."` if `stored_ptr == ptr`.

Reply:
That's like re-implementing peek, that doesn't smell right. If I'd want to change this, I'd look for ways to structure the code so that I could just eat the whitespace, because there's no grammar structure that doesn't allow it.

But then I didn't want to invest my brain in that, too.